mlir/lib/Dialect/Vector/Transforms/LowerVectorContract.cpp

```diff
@@ -1302,13 +1304,18 @@ FailureOr<Value> ContractionOpToMatmulOpLowering::matchAndRewriteMaskableOp(
       !isReductionIterator(iteratorTypes[2]))
     return failure();
 
+  Type opResType = op.getType();
+  VectorType vecType = dyn_cast<VectorType>(opResType);
+  if (vecType && vecType.isScalable()) {
+    // This should be sufficient to reject all cases with scalable vectors.
```

Comment:
nit: is the ambiguity is necessary, surely this is sufficient to reject all cases with scalable vectors?